src/Aspire.Hosting.AWS/AWSSDKConfig.cs

```diff
@@ -11,4 +11,10 @@ internal sealed class AWSSDKConfig : IAWSSDKConfig
 
     /// <inheritdoc/>
     public RegionEndpoint? Region { get; set; }
+
+    /// <summary>
+    /// Sets if the sdk will validate the profile.
+    /// </summary>
+    internal bool DisableSdkValidation { get; set; }
```

Comment:
i wonder if we should add this to the IAWSSDKConfig interface? And then update this to be public and also have ///inheritdoc/> similar to RegionEndpoint

Reply:
Also i think we should make this the inverse to make it easier to read - SDKValidationEnabled  and have the default value as `true`